jgiven-html-app/src/test/utilTest.js

```diff
@@ -271,4 +271,35 @@ describe("Util", function () {
           expect(replaceArguments("$2, $unknown, $", enumArray, nameArray)).toEqual("10, falseunknown, 10");
       });
   });
+
+  describe("substituteThumbnailPath", function() {
+      it("works for paths with no numbers", function() {
+           var path0 = "/simple/data/path.png";
+           expect(insertThumbnailPath(path0)).toEqual("/simple/data/path-thumb.png")
+      });
+
+      it("works for paths with single number", function() {
+           var path0 = "/simple/data/path0.png";
+           expect(insertThumbnailPath(path0)).toEqual("/simple/data/path-thumb0.png");
```

Comment:
this would mean that the `getTargetFile` method from `Html5AttachmentGenerator`would have to be duplicated for thumbnails, mainly because of the enumeration strategy via multisets